stdlib/LinearAlgebra/test/matmul.jl

```diff
@@ -425,7 +425,8 @@ end
     @test  transpose(Xv2)*Xv2     ≈ XtX
     @test (transpose(Xv3)*Xv3)[1] ≈ XtX
     @test  Xv1'*Xv1     ≈ XcX
-    @test_throws MethodError Xv2'*Xv2 ≈ XcX
+    @test_throws MethodError Xv2'*Xv2 ≈ XcX # Xv2'*Xv2 is a scalar, XcX a matrix
+    @test Xv2'*Xv2 ≈ norm(Xv2)^2
```

Comment:
Given that we're already testing that the result is approximately equal to a scalar, this is more of a test of the behavior of `≈` which isn't necessary (and doesn't really belong here).

Reply:
I've removed the test.